tests/event-processing-tests/src/test/java/uk/gov/di/txma/eventprocessor/bdd/LambdaToS3StepDefinitions.java

```diff
@@ -702,4 +724,51 @@ public void theEventHasBeenAdded(String eventName) {
         JSONObject JSONWithEventAdded = addEventName(rawJSON, eventName);
         lambdaInput = wrapJSONObjectAsAnSQSMessage(JSONWithEventAdded);
     }
+
+    @Given("the SQS data file {string} is available for the {string} team")
+    public void the_sqs_data_file_is_available_for_the_team(String fileName, String account) throws IOException {
+        // Write code here that turns the phrase above into concrete actions
+        JSONObject rawJSON = new JSONObject(readJSONFile(fileName));
+        JSONObject enrichedJSON = addComponentIdAndTimestampFields(rawJSON, account);
+        lambdaInput = wrapJSONObjectAsAnSQSMessage(enrichedJSON);
+    }
+
+
+
+    public static void receiveSqsMessage(String CFN_SqsURL, int timeout) {
+        SqsClient sqs = SqsClient.builder().build();
+
+        // Get the receipt handle for the first message in the queue
+        ReceiveMessageRequest receiveRequest = ReceiveMessageRequest.builder()
+                .queueUrl(CFN_SqsURL)
+                .build();
+        Message message = sqs.receiveMessage(receiveRequest)
+                .messages()
+                .get(0);
+        sqsoutputtext = message.body();
+        DeleteMessageRequest deleteMessageRequest = DeleteMessageRequest.builder()
+                .queueUrl(CFN_SqsURL)
+                .receiptHandle(message.receiptHandle())
+                .build();
+        sqs.deleteMessage(deleteMessageRequest);
+        System.out.println("sqsoutput " +sqsoutputtext);
```

Comment:
Might be worth removing this

Reply:
This is required to ensure we delete the message after processing.